app/assets/stylesheets/forever_style_guide/globals/_fonts.scss

```diff
@@ -25,22 +27,52 @@ $font-face-regular:         'proxima_nova-regular', Helvetica, Arial, sans-serif
 $font-face-semibold:        'proxima_nova-semibold', Helvetica, Arial, sans-serif;
 $font-face-bold:            'proxima_nova-bold', Helvetica, Arial, sans-serif;
 $font-face-regular_italic:  'proxima_nova-regular_italic', Helvetica, Arial, sans-serif;
-$font-face-bold_italic:     'proxima_nova-bold_italic', Helvetica, Arial, sans-serif;;
+$font-face-bold_italic:     'proxima_nova-bold_italic', Helvetica, Arial, sans-serif;
 
 $font-face-gotham:          'Gotham A', 'Gotham B', Helvetica, Arial, sans-serif;
 
 /* =font sizes
    ---------------------------------------------------------------------------- */
-$font-size-100: 12px;
-$font-size-200: 14px;
-$font-size-300: 16px;
-$font-size-400: 18px;
-$font-size-500: 24px;
-$font-size-600: 36px;
-$font-size-700: 48px;
-$font-size-800: 64px;
-
-$font-size-large: $font-size-400;
-$font-size-default: $font-size-300;
-$font-size-small: $font-size-100;
+$font-size-100: fontify('100', 12px) !default;
+$font-size-200: fontify('200', 14px) !default;
+$font-size-300: fontify('300', 16px) !default;
+$font-size-400: fontify('400', 18px) !default;
+$font-size-500: fontify('500', 24px) !default;
+$font-size-600: fontify('600', 36px) !default;
+$font-size-700: fontify('700', 48px) !default;
+$font-size-800: fontify('800', 64px) !default;
+
+$font-size-large: fontify('large', $font-size-400) !default;
+$font-size-default: fontify('default', $font-size-300) !default;
+$font-size-small: fontify('small', $font-size-100) !default;
+
+@each $id, $font-size in $font-sizes {
+  .font-size-#{$id} {
+    font-size: #{$font-size};
+  }
+}
+
+/* =font weights
+   ---------------------------------------------------------------------------- */
+$font-weight-100: weightify('100', 100) !default;
```

Comment:
Two questions here:
* I don't think these classes will work w/ Proxima, so should we add a comment here, or implement otherwise?
* And then below do you think using the weight by name (ie lighter) is preferable to using classes w/ numbers since those aren't relative to a given parent?